torch/onnx/symbolic_helper.py

```diff
@@ -521,37 +521,63 @@ def _get_dim_for_cross(x: _C.Value, dim: Optional[int]):
     return dim
 
 
-def _unimplemented(op: str, msg: str):
+def _unimplemented(op: str, msg: str, value: Optional[_C.Value] = None) -> None:
```

Comment:
```suggestion
def _unimplemented(op: str, msg: str, value: Optional[_C.Value] = None) -> Union[None, SymbolicValueError]:
```

Reply:
For Caffe2 we just issue a warning and returns `None` otherwise, `UnsupportedOperatorError` should be returned (previously it was `RuntimeError`

Reply:
When an error is raised nothing is returned so we still have None as the return type

Reply:
Mark as resolved.